components/FullSiteArticleStats.js

```diff
@@ -5,16 +5,41 @@ import React from 'react';
  * @param {number} props.stats.repliedCount
  * @param {number} props.stats.notRepliedCount
  */
-function FullSiteArticleStats({ stats }) {
+function FullSiteArticleStats({ stats, repliedArticleCount }) {
   if (!stats) return null;
 
   return (
     <div className="full-site-stat">
       全站未回訊息： {stats.get('notRepliedCount')}
+      <p>總篇數: {stats.get('repliedCount') + stats.get('notRepliedCount')}</p>
+      <p>
+        他人回復的: {stats.get('repliedCount') - repliedArticleCount} /
+        你回復的: {repliedArticleCount}
```

Comment:
Please change 回復 to 回應 because
- Most places in this site uses "回應"
- We are not healing anyone or restoring anything so "回復" is not appropriate here